core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -42,6 +42,7 @@ <h2 class="oppia-preferences-page-heading-subtext">
             </div>
             <i class="fas fa-pen oppia-editor-edit-icon oppia-editor-edit-icon-pos"
                tabindex="0"
+               aria-label="Edit-button"
```

Comment:
I removed line number 44 (extra i tag) and modified the existed i tag, added aria-label to it.

Reply:
(just out of curiosity) @seanlip shouldn't the aria-label be localised?

Reply:
Actually, yes ... but we haven't figured out that part, I think. So I would say: if other aria-labels are not translated, let's not translate this one, and file a separate issue for that. If they are, then this one should be localized the same way.

Reply:
Hi, @seanlip do I need to do any other kind of changes in this file.

Reply:
Please change the aria label to "Edit profile picture", since that's what the button does. Otherwise I think it looks fine, thanks.